bpf/lib/auth.h

```diff
@@ -10,17 +10,30 @@
 #include "signal.h"
 
 static __always_inline int
-auth_lookup(struct __ctx_buff *ctx, __u32 local_id, __u32 remote_id, __u16 remote_node_id,
+auth_lookup(struct __ctx_buff *ctx, __u32 local_id, __u32 remote_id, __u32 remote_node_ip,
 	    __u8 auth_type)
 {
+	struct node_key node_ip = {};
 	struct auth_info *auth;
 	struct auth_key key = {
 		.local_sec_label = local_id,
 		.remote_sec_label = remote_id,
-		.remote_node_id = remote_node_id,
 		.auth_type = auth_type,
 		.pad = 0,
 	};
+	__u16 *node_id;
+
+	if (remote_node_ip) {
+		node_ip.family = ENDPOINT_KEY_IPV4;
```

Comment:
just for clarification: the tunnel IP is always IPv4?

Reply:
Yep, that's correct.